MobileCenter/MobileCenter/Internals/Utils/MSUtility+Application.h

```diff
@@ -0,0 +1,44 @@
+#import <Foundation/Foundation.h>
+#import "MSUtility.h"
+
+/**
+ *  App states
+ */
+typedef NS_ENUM(NSInteger, MSApplicationState) {
+    
+    /**
+     * Application is active.
+     */
+    MSApplicationStateActive = UIApplicationStateActive,
+    
+    /**
+     * Application is inactive.
+     */
+    MSApplicationStateInactive = UIApplicationStateInactive,
+    
+    /**
+     * Application is in background.
+     */
+    MSApplicationStateBackground = UIApplicationStateBackground,
+    
+    /**
+     * Application state can't be determined.
+     */
+    MSApplicationStateUnknown
+};
+
+/**
+ * Utility class that is used throughout the SDK.
+ * Application part.
+ */
+@interface MSUtility (Application)
+/**
+ * Get current application state.
+ *
+ * @discussion The application state may not be available anywhere. Application extensions doesn't have it for instance,
```

Comment:
Move `@discussion` below `@return`. You also need to add an empty line between different annotations.